iModelCore/ECDb/ECDb/ECSql/ECSqlPreparer.h

```diff
@@ -6,6 +6,7 @@
 
 #include "ECSqlParser.h"
 #include "ECSqlPrepareContext.h"
+#include "WindowFunctionExp.h"

```

Comment:
Should not need this since ECSqlParser is included which includes WindowFunctionExp

Reply:
removed